Error: cannot find value `VERSION_3_5_1` in crate `protobuf`

There is a compilation error when installing `imessage-exporter` from `cargo`:

```rust
error[E0425]: cannot find value `VERSION_3_5_1` in crate `protobuf`
  --> /Users/chris/.cargo/registry/src/index.crates.io-6f17d22bba15001f/imessage-database-2.1.0/src/message_types/handwriting/handwriting_proto.rs:25:49
   |
25 | const _PROTOBUF_VERSION_CHECK: () = ::protobuf::VERSION_3_5_1;
   |                                                 ^^^^^^^^^^^^^ help: a constant with a similar name exists: `VERSION_3_6_0`
   |
  ::: /var/folders/yv/ss1j98_x729f58qy7xlrxggc0000gn/T/cargo-installYuV2y3/release/build/protobuf-ca62339f43aef977/out/version.rs:7:1
   |
7  | pub const VERSION_3_6_0: () = ();
   | --------------------------- similarly named constant `VERSION_3_6_0` defined here

For more information about this error, try `rustc --explain E0425`.
error: could not compile `imessage-database` (lib) due to 1 previous error
error: failed to compile `imessage-exporter v2.1.0`, intermediate artifacts can be found at `/var/folders/yv/ss1j98_x729f58qy7xlrxggc0000gn/T/cargo-installYuV2y3`.
```

This happens because `cargo` is installing version `3.6.0` of `protobuf`:

<img width="285" alt="image" src="https://github.com/user-attachments/assets/99743d09-e8d7-4fae-9120-d412d9f16dd9">

However, the `cargo.toml` requires `3.5.1`:

https://github.com/ReagentX/imessage-exporter/blob/e6b45b910efe47d206b9e761ed57fd3807812269/imessage-database/Cargo.toml#L17

--- a/imessage-database/Cargo.toml
+++ b/imessage-database/Cargo.toml
@@ -10,13 +10,13 @@ repository = "https://github.com/ReagentX/imessage-exporter"
 version = "0.0.0"
 
 [dependencies]
-chrono = "0.4.38"
-plist = "1.7.0"
-rusqlite = { version = "0.32.1", features = ["blob", "bundled"] }
-sha1 = "0.10.6"
-protobuf = "3.5.1"
-lzma-rs = "0.3.0"
+chrono = "=0.4.38"
+plist = "=1.7.0"
+rusqlite = { version = "=0.32.1", features = ["blob", "bundled"] }
+sha1 = "=0.10.6"
+protobuf = "=3.5.1"
+lzma-rs = "=0.3.0"
 
 [build-dependencies]
-protobuf = "3.5.1"
-protobuf-codegen = "3.5.1"
+protobuf = "=3.5.1"
+protobuf-codegen = "=3.5.1"

--- a/imessage-exporter/Cargo.toml
+++ b/imessage-exporter/Cargo.toml
@@ -10,10 +10,10 @@ repository = "https://github.com/ReagentX/imessage-exporter"
 version = "0.0.0"
 
 [dependencies]
-clap = { version = "4.5.11", features = ["cargo"] }
-filetime = "0.2.23"
-fdlimit = "0.3.0"
-fs2 = "0.4.3"
+clap = { version = "=4.5.11", features = ["cargo"] }
+filetime = "=0.2.23"
+fdlimit = "=0.3.0"
+fs2 = "=0.4.3"
 imessage-database = { path = "../imessage-database" }
-indicatif = "0.17.8"
+indicatif = "=0.17.8"
 rusqlite = { version = "0.32.1", features = ["blob", "bundled"] }

